app/views/api2/_image.xml.builder

```diff
@@ -25,13 +25,13 @@ xml.tag!(
     xml_minimal_object(xml, :owner, :user, object.user_id)
   else
     xml_detailed_object(xml, :owner, object.user)
-    xml.files(number: Image.all_sizes.count + 1) do
+    xml.files(number: Image.all_sizes.size + 1) do
```

Comment:
Is `length` more performant here? 
>If the collection has been already loaded `size` and `length` are equivalent. If not and you are going to need the records anyway `length` will take one less query. Otherwise `size` is more efficient.

https://api.rubyonrails.org/classes/ActiveRecord/Associations/CollectionProxy.html#method-i-size